ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -113,6 +113,14 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-ovn
+- always_run: false
```

Comment:
@anuragthehatter you need to import golang image
currently it is 4.16
```yaml
  tests-private:
    name: tests-private
    namespace: ci
    tag: "4.16"
```

Reply:
@kuiwang02 PTAL for final changes. Thanks

Reply:
@anuragthehatter for 
```yaml
  tests-private:
    name: tests-private
    namespace: ci
    tag: latest
```
it should be in base part, like
```yaml
base_images:
  dev-scripts:
    name: test
    namespace: ocp-kni
    tag: dev-scripts
  hypershift-operator:
    name: "4.16"
    namespace: ocp
    tag: hypershift-operator
  ocp_4.16_base-rhel9:
    name: "4.16"
    namespace: ocp
    tag: base-rhel9
  ocp_builder_rhel-8-golang-1.21-openshift-4.16:
    name: builder
    namespace: ocp
    tag: rhel-8-golang-1.21-openshift-4.16
  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
    name: builder
    namespace: ocp
    tag: rhel-9-golang-1.21-openshift-4.16
  tests-private:
    name: tests-private
    namespace: ci
    tag: latest
```

Note: before submitting the code, pelase run `make update` in your local.